docs/static/managing-multiline-events.asciidoc

```diff
@@ -5,10 +5,17 @@ Several use cases generate events that span multiple lines of text. In order to
 Logstash needs to know how to tell which lines are part of a single event.
 
 Multiline event processing is complex and relies on proper event ordering. The best way to guarantee ordered log
-processing is to implement the processing as early in the pipeline as possible. The preferred tool in the Logstash
-pipeline is the {logstash}plugins-codecs-multiline.html[multiline codec], which merges lines from a single input using
-a simple set of rules.
+processing is to implement the processing as early in the pipeline as possible.
+
+IMPORTANT: If you are using a Logstash input plugin that supports multiple hosts, such as
+the <<plugins-inputs-beats>> input plugin, you should not use the
+<<plugins-codecs-multiline>> codec to handle multiline events. Doing so may result in the 
+mixing of strings and corrupted event data. In this situation, you need to handle multiline
+events before sending the event data to Logstash. 
 
+The <<plugins-codecs-multiline>> codec is the preferred tool for handling multiline events
+in the Logstash pipeline. The multiline codec merges lines from a single input using
```

Comment:
Yah I was suggesting only because there are users who don't read the documentation line by line, so if somehow they missed the important section, then they may think that multiline codec applies to all.  But we can just leave it as-is, since we already have the ticket open to block users from even attempting to configure the beats input with ML codec :)  What if we simply move the Important section to be right below the "preferred tool  ..." paragraph instead of above it?